webapp/src/components/card/subscription/index.tsx

```diff
@@ -97,6 +97,14 @@ const SubscriptionCard = ({handleDeleteSubscrption, subscriptionDetails: {channe
                                     {approvalStatusName && <Chip text={`Approval status is: ${approvalStatusName}`}/>}
                                     {approvalTypeName && <Chip text={`Approval type is: ${approvalTypeName}`}/>}
                                     {releaseStatusName && <Chip text={`Release status is: ${releaseStatusName}`}/>}
+                                    {runPipelineName && <Chip text={`Pipeline is: ${runPipelineName}`}/>}
+                                    {runEnvironment && <Chip text={`Environment is: ${runEnvironment}`}/>}
+                                    {runStage && <Chip text={`Stage is: ${runStage}`}/>}
+                                    {runStageId && <Chip text={`Stage is: ${runStageId}`}/>}
```

Comment:
```suggestion
                                    {runStageId && <Chip text={`Stage Id is: ${runStageId}`}/>}
```

Reply:
The display name for this is actually "Stage" in the UI

Reply:
One more chip is present above with the same data `Stage is:`. will the user be able to differentiate between them?

Reply:
Yes, they both can not exist at the same time

Reply:
they are for two different event types